docs/BEMAN_STANDARD.md

```diff
@@ -112,27 +113,35 @@ to build it, and links to further documentation.
 
 ## `README.md`
 
-**[README.TITLE]** RECOMMENDATION: The `README.md` should begin with a level 1
-header with the name of the library optionally followed with a ":" and short
-description.
+**[README.TITLE]** RECOMMENDATION: The `README.md` must begin with a level 1 header with the name of the library optionally followed with a ":" and short description.
 
 Examples:
 
 ```markdown
 # beman.sender_receiver: Scalable Asychronous Program Building Blocks
 ```
 
-**[README.PURPOSE]** RECOMMENDATION: Following the title, the `README.md` should
-contain a one- or two-paragraph summary describing the library's purpose.
+**[README.PURPOSE]** REQUIREMENT: Following the title, the `README.md` must contain a one or two-paragraph summary describing the library's purpose.
 
-**[README.IMPLEMENTS]** RECOMMENDATION: Following the purpose and a newline, the
-`README.md` should indicate which papers the repository implements. Use the following style:
+**[README.IMPLEMENTS]** REQUIREMENT: Following the purpose and a newline, the `README.md` must indicate which papers the repository implements. Use the following style:
 
 ```markdown
-**Implements:** [`std::optional<T&>` (P2988R5)](https://wg21.link/P2988R5) and
+**Implements**:[`std::optional<T&>` (P2988R5)](https://wg21.link/P2988R5) and
 [Give *std::optional* Range Support (P3168R1)](https://wg21.link/P3168R1).
 ```
 
+or, if this is an early idea implementation and there is no paper available:
+
+```markdown
+**Implements**: N/A
+```
+
+**[README.LIBRARY_STATUS]** REQUIREMENT: Following the implements section and a newline, the `README.md` must indicate the library status. Check [BEMAN_PROCESS.md#beman-libraries-lifetime](./BEMAN_PROCESS.md#beman-libraries-lifetime). Use the following style:
+
+```markdown
+**Library status**: [BEMAN STABLE](./BEMAN_PROCESS.md#beman-libraries-lifetime) (since (#abcdef)[https://github.com/bemanproject/optional26/commit/#abcef] - 20.11.2024).
```

Comment:
That it's the library status is implied...

```suggestion
**Status**: [BEMAN STABLE](./BEMAN_PROCESS.md#beman-libraries-lifetime) (since (#abcdef)[https://github.com/bemanproject/optional26/commit/#abcef] - 20.11.2024).
```

Instead of using a special parlance, I suggest using one of the more descriptive labels I suggested above. Something like this:

```suggestion
**Status**: [Production ready. API may undergo changes.](./BEMAN_PROCESS.md#production-ready-api-may-undergo-changes) (since (#abcdef)[https://github.com/bemanproject/optional26/commit/#abcef] - 20.11.2024).
```

I also think the "since..." is not useful for most people. A change log is a good idea, but that belongs somewhere else.

```suggestion
**Status**: [Production ready. API may undergo changes.](./BEMAN_PROCESS.md#production-ready-api-may-undergo-changes)
```